public/js/reader.js

```diff
@@ -111,33 +111,27 @@ Reader.loadImages = function () {
             if (Reader.infiniteScroll) {
                 Reader.initInfiniteScrollView();
             } else {
-                $(window).on('wheel', function(e) {
+                $(window).on("wheel", function(e) {
                     if(Reader.isFullscreen) {
-                        if (e.originalEvent.deltaY > 0) Reader.changePage(-1);
-                        else Reader.changePage(1);
+                        let changePage_val = 1;
+                        if (e.originalEvent.deltaY > 0) changePage_val = -1;
+                        if (!Reader.mangaMode) changePage_val *= -1;
```

Comment:
An extra comment'd be nice here:  
```suggestion
                        // In Manga mode, reverse the changePage variable so that we always move forward
                        if (!Reader.mangaMode) changePage_val *= -1;
```